MiniApp/ja.lproj/Localizable.strings

```diff
@@ -2,6 +2,6 @@
 "Cancel_title" = "キャンセル";
 
 // MARK: - UI
-"ui_close_button_title" = "Close";
-"ui_loading_failed_error" = "An error has occured.\nPlease try again";
-"ui_retry_button_title" = "Retry";
+"ui_close_button_title" = "クローズ";
+"ui_loading_failed_error" = "情報を取得できませんでした。\n再度お試しください。";
```

Comment:
we should make this text reviewed by another Japanese speaking person. The meaning is a bit different from the original message.

Reply:
sure I've just copied it from Link except close 😄

Reply:
@Climbatize to whom can we reach out for the translations or should I just keep english for now?

Reply:
I think we can keep just the english for now - as long as we make sure the strings can be localized by the Host App. But I think @Climbatize you will work on some changes to make strings in the SDK localizable?

Reply:
yes that's true @corycaywood, I'll take in charge all these localization concerns

Reply:
@Climbatize ok I'll comment out the japanese localization for now (using your localization imp later)